poster/poster.tex

```diff
@@ -181,20 +181,22 @@ \section*{Introduction}
 be opened in the whole Ginga session, whereas a local plugin is
 associated with the channel it is started from: one instance can be
 opened per channel and different instances can be configured separately
-in the same Ginga session. 
-Plugins are currently not available when Ginga is used as part of Jupyter
-Notebook/Lab. However, they are usable in a limited way via Ginga's remote
-control (RC) interface, which will not be covered here.
+in the same Ginga session.
+%% Plugins are currently not available when Ginga is used as part of Jupyter
+%% Notebook/Lab. However, they are usable in a limited way via Ginga's remote
+%% control (RC) interface, which will not be covered here.
 
 \para
 Currently, all the plugins in {\tt stginga} are local plugins.
-As with other Ginga plugins, they can be customized via
+As with many Ginga plugins, they can be customized via a file in the
+user's home directory:
 {\tt \textasciitilde/.ginga/plugin\_{\em PluginName}.cfg}.
 Some plugins currently in Ginga originated from {\tt stginga}
 (e.g., {\em ChangeHistory}, {\em SaveImage}, {\em TVMark}, and {\em TVMask})
 when they were identified to be useful in general beyond HST or JWST.
-All the screenshots shown in this poster used Qt5 backend, although they should
-also work in Qt4, GTK 2, and GTK 3.
+The viewer can use different toolkit \emph{backends};
+all the screenshots shown in this poster used Qt5 backend, although they should
+also work with the Qt4, PySide, Gtk 2 and Gtk 3 backends.
```

Comment:
* I thought GTK is all capital letters? Reference: https://www.gtk.org/
* Nitpick: Oxford comma after GTK 2.

Reply:
ok, if that is considered correct capitalization.  I'm ok with oxford commas. I was just trained the other way!